flasher.cpp

```diff
@@ -255,8 +266,50 @@ void Flasher::LoopHandler()
             break;
         }
 
+        case FlasherStates::kLoadFirmwareFile:
+        {
+            // Load local firmware file
+            if (firmware_file_.size() != 0) {
+                file_content_ = firmware_file_.readAll();
+                firmware_file_.close();
+                firmware_file_.remove();
+                SetState(FlasherStates::kFlash);
+
+                // Load firmware file from url
```

Comment:
Maybe we can put this comment inside else statement. It will look nicer, at least to me.